charmcraft.yaml

```diff
@@ -16,13 +16,20 @@ bases:
     - name: ubuntu
       channel: "22.04"
 
-# config:
-#   options:
-#     snap_channel:
-#       default: "latest/stable"
-#       type: string
-#       description: |
-#           If install_method is set to "snap" this option controls channel name.
+requires:
+  credentials:
+    interface: keystone-admin
```

Comment:
question: what is the difference between keystone-admin and keystone-credential?

Reply:
oh, side note: I was just wanting to leave a comment, but the status become request-change